com.github.robertsanseries.ciano.json

```diff
@@ -91,30 +126,37 @@
             "post-install": [
                 "chrpath -d /app/bin/ffmpeg"
             ],
-            "sources": [
-                {
-                  "type": "archive",
-                  "url": "https://ffmpeg.org/releases/ffmpeg-4.4.tar.xz",
-                  "sha256": "06b10a183ce5371f915c6bb15b7b1fffbe046e8275099c96affc29e17645d909"
-                }
-            ]
+            "sources": [{
+                "type": "archive",
+                "url": "https://ffmpeg.org/releases/ffmpeg-7.0.2.tar.xz",
+                "sha256": "8646515b638a3ad303e23af6a3587734447cb8fc0a0c064ecdb8e95c4fd8b389"
+            }]
```

Comment:
also these changes. There shuld only be 2 lines being changed.

Reply:
In some circumstances I find it more readable. But you're right, do you want me to return everything to the previous format?